spec/models/status_message_spec.rb

```diff
@@ -63,48 +61,45 @@
   end
 
   describe ".guids_for_author" do
-    it 'returns an array of the status_message guids' do
-      sm1 = FactoryGirl.create(:status_message, :author => alice.person)
-      sm2 = FactoryGirl.create(:status_message, :author => bob.person)
+    it "returns an array of the status_message guids" do
+      status_message_1 = FactoryGirl.create(:status_message, author: alice.person)
+      status_message_2 = FactoryGirl.create(:status_message, author: bob.person)
```

Comment:
Useless assignment to variable - `status_message_2`. Did you mean `status_message_1`?